src/deriver/queue_manager.py

```
@@ -116,6 +116,15 @@ async def initialize(self) -> None:
         """Setup signal handlers, initialize client, and start the main polling loop"""
         logger.debug(f"Initializing QueueManager with {self.workers} workers")
 
+        recovered_dreams = 0
+        try:
+            recovered_dreams = await self.dream_scheduler.recover_overdue_dreams()
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)